MatrixSDK/JSONModels/MXEvent.m

```diff
@@ -649,6 +650,10 @@ - (MXEvent*)prune
 
 - (MXEvent*)editedEventFromReplacementEvent:(MXEvent*)replaceEvent
 {
+    if ([self.eventId isEqualToString:replaceEvent.eventId]) {
```

Comment:
Does that actually happen? I thought it wouldn't be possible to send an event that replaces itself as the server generates the event ID after you've sent it.

Reply:
I should actually remove this, it got left behind from when I was trying to refactor some things.